channels/email/settings.rst

```diff
@@ -22,14 +22,34 @@ Notification Sender: Default value ``Notification Master <noreply@#{config.fqdn}
    This address is relevant for agent notifications and password reset mails
    (also affects customers).
 
-Additional follow-up detection
-   In some situations the normal follow-up detection is not enough.
-   This might be due to missing references in the subject
-   (the ticket hook and number). These options can help to recognize follow-ups
-   to existing tickets.
-
-   Please note that searching in attachment and body might lead to *false
-   follow-up detection*.
+Additional follow-up detection: Default value ``Subject & References``
+   Adjust how Zammad checks if an incoming email is a follow-up or not.
+   If you disable all checks, Zammad only checks if a matching ticket hook &
+   number is present in the subject.
+
+   Subject & References
+      Checks subject and message IDs of emails and searches for initial ticket
+      articles with the same subject and message ID. When both are matching,
+      it is considered as a follow-up.
+
+   References
+      It only checks if a message ID from email matches an existing article.
```

Comment:
It's checked in addition to the default subject ticket hook & number check. Sorry, I thought that was clear. Just reformulate the start of the sentence like those below, and it should be fine.